Tools/CISupport/ews-build/steps.py

```diff
@@ -623,14 +627,17 @@ def run(self):
         remote = self.getProperty('github.head.repo.full_name', 'origin').split('/')[0]
         project = self.getProperty('github.head.repo.full_name', self.getProperty('project'))
         branch = self.getProperty('github.head.ref', 'main')
+        base = self.getProperty('github.base.sha')
+        rebase_target = self.getProperty('ews_revision') or self.getProperty('got_revision')
 
         for command in [
+            ['git', 'config', 'merge.changelog.driver', 'perl Tools/Scripts/resolve-ChangeLogs --merge-driver -c %O %A %B'],
```

Comment:
We only need this once per checkout. For reference, engineers set this with `git-webkit setup`. I thought about doing this in `CheckoutSource`, but that would mean that machines which already had checkouts wouldn't get it. This command is very quick to run, though, so I thought it best to have it run here since we immediately use it 6 lines down. To address your "better readability" comment, if I'm going to make `commands` into a variable, I also plan on tying the `merge` to the `rebase` more explicitly.